hc/accounts/tests/test_profile.py

```diff
@@ -21,22 +21,30 @@ def test_it_sends_set_password_link(self):
         self.assertTrue(len(token) > 10)
 
         ### Assert that the token is set
+        self.assertTrue(token)
 
         ### Assert that the email was sent and check email content
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertIn("Here's a link to set a password for your account on My Monitoring Project:", mail.outbox[0].body)
+
+
 
     def test_it_creates_api_key(self):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"create_api_key": "1"}
         r = self.client.post("/accounts/profile/", form)
+        
         ### Test it creates API key
+        self.assertTrue(r)
 
     def test_it_revokes_api_key(self):
         self.client.login(username="alice@example.org", password="password")
 
         form = {"revoke_api_key": "1"}
         r = self.client.post("/accounts/profile/", form)
```

Comment:
this has been changed to 'response' on new commit.
please note that many of the  tests were not created by us and may still contain wrong variable names.